### DISCUSSION TITLE
Need help removing the recommended section from windhawk startmenu

### DISCUSSION THREAD

**Discussion by awaixjvd** [ramensoftware/windhawk-mods#1279] [General]
I am new to this windhawk and don't know coding. I am using windows 11 start menu styler and style being used is "sidebyside2" and i really love it. I just want to remove the recommended section from it. Does anyone know code to remove it?

I tried some other menus but didn't like it. I also tried some codes from VS themes site but they are very rounded and i don't know which code line to change. So i want to stick to default "sidebyside2" and just remove the recommended section.

**Comment by m417z** [comment]
Answered on Discord. Posting the answer here for reference:

> Import the NoRecommendedSection theme manually using the instructions here:
> https://github.com/ramensoftware/windows-11-start-menu-styling-guide/blob/main/Themes/NoRecommendedSection/README.md
> Then go to the mod settings and select the SideBySide2 theme.
